### ISSUE TITLE
Suggestion: Merging Inventory Instead of Replacing When Player Login

### ISSUE THREAD

**Issue by leavessoft** [brunyman/MysqlInventoryBridge#4]
**Given this situation:** There are two servers, one of them has just cleared all the players' inventories while the other didn't. And this plugin was freshly installed after the clearing.
Then a player entered the first server (whose player data was deleted), which means after he/she entering the second server above, he/she will lose all his items in the inventory.

**Comment by brunyman** [comment]
Probably the best option for this case is an import data command, this way you import player's inventory from the server you need to. Else merging open's the gates for a possible dupe.

**Comment by leavessoft** [comment]
> Probably the best option for this case is an import data command, this way you import player's inventory from the server you need to. Else merging open's the gates for a possible dupe.

Yep. Could options be shown after players joining the server when ambiguous exists, allowing them to select which inventory they want to keep?
